src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-darPath.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Path to dar file(relative to workspace) which needs to be published on xl-deploy server.
```

Comment:
Path to the DAR file (relative to the workspace) that should be published on the XL Deploy server.